tests/unit/test_prometheus.py

```diff
@@ -55,7 +54,7 @@ def test_prometheus_metrics_config():
                                                 start_now=True)
     assert prometheus_config.listen_address == listen_address
     assert prometheus_config.collection_interval == collection_interval
-    assert prometheus_config.start_now
+    assert prometheus_config.start_now is True
```

Comment:
Just curious, why do we need to `is True` here? Wouldn't a plain `assert` work?

Reply:
Yes, the plain assertion was working perfectly. I just thought that a more explicit assertion could be verbose and, hence, more clear about what we are checking here.

But I don't have a strong opinion here, we can return to the original assertion if that works better.

Reply:
I don't have a strong opinion either, `assert` is taking both values and expressions and a sprinkle of verbosity wouldn't hurt